### PR TITLE
Watch video page

### PR DESCRIPTION
**Pull request by Lameck1**
# Watch video page

### Description

This pull request submits features of the watch video page for review and merging

##### Page Sections Completed

- Header
- Section
- Aside
- Footer

Provided below is the screenshot of the Watch Video Page

home-page

### CODE REVIEW

File: styles/styles.css
Comment (teekaytech) [inline styles/styles.css]: You used nested CSS selectors here. This is inefficient. Kindly re-factor your code by using class names to select elements. Whenever you find yourself nesting more than 2 selectors, it's getting out of hand.

t1

Check out [this link](https://speckyboy.com/good-bad-css-practices) for best CSS practices.

**Review by teekaytech — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle: :vertical_traffic_light: 

### Hello team, congratulations on successful submission of your first-ever code review request since joining the program:clap:. You guys have put in a great deal of effort in building this project. :muscle: 

**However, there are some changes that must be made before this project is approved. Kindly make the changes as highlighted below and request for another review afterward.**

* * *

**[REQUIRED]** _these changes must be made_

- [ ] Your page should have at least 3 images/icons at the rightmost part of the navbar. Right now, you're having 2 icons. You can add a user avatar to complete it.

t2

- [ ] For this project, you are not allowed to use screenshots of the original page as you've done in the sidebar recommended videos section. It's not as difficult as it seems. Try to build this by yourself.

t3

- [ ] Your page is having more blank space to the leftmost part than the rightmost part. Please try to improve on this.

y

- [ ] Check the inline comment I made below and fix as appropriate.


**[OPTIONAL]**

- [ ] Improve the video information section of your page. 
- You can make the video title bigger/bolder. 
- Your subscribe button can also be improved.
- Try to make your layout look like the original.
**Your page** 
y

**Original page**
s

- [ ] Always include your project's live demo link in your readme. [This website](https://raw.githack.com/) might help.

- [ ] You are encouraged to implement the comments section of this page, as seen in the original

t5


**[COMMENTS]**

- [ ] In your next projects, try to be more descriptive in your commit messages. Some of your commit messages are not descriptive, most especially the ones created by @ArreyTabe.

**Please fix the above changes required and request another review when you're done.**

* * *
**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**

_Dear team, do not be discouraged. We're here to help you and we believe these changes make your project more awesome._ 
**Happy Coding :man_technologist:**

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required

**Great work! the project has the main elements with their details, also see that you corrected the previous observations made.**

This things are good to notice:
:100: Descriptive README. (see optional requirements to see how it could be improved)
:100: Descriptive Pull Request and Commit messages
:100: On the effort your putting in this project.

### [Mandatory]
- [x] Fix linter errors. Project not passing all linter checks.

- [x] Navbar overlaping content.
image

### [OPTIONAL]
- [ ] Add link to live preview in the README. [Live preview generator](https://rawcdn.githack.com)

## Keep the great work :muscle: , and Happy coding :smiley: :computer: !

**Review by jubaan — APPROVED** [review]
# Status: Approved!!! :tada: :tada:

I see you fix all the remaining issues!
:heavy_check_mark: Passing all linter checks.
:heavy_check_mark:  Corrected overlapping.
:heavy_check_mark: Add live preview link to README.

## :tada: Contrats :party: :balloon: !!!

**Pull request merged**
